src/scripts/SpellHandlers/PaladinSpells.cpp

```diff
@@ -366,6 +366,17 @@ bool JudgementOfTheWise(uint32 i, Spell* s)
     return true;
 }
 
+bool GuardedByTheLight(uint32 i, Spell* s)
+{
+	if (!s->p_caster)
```

Comment:
I am not sure what you mean. The condition already has braces around it. Do you mean to put the return in braces like this:

if (!s->p_caster)
{
    return false;
}

Cause most of the ifs in the same file are not written like this.